src/extends/room/control.js

```diff
@@ -16,7 +16,7 @@ let roomLevelOptions = {
     'DEDICATED_MINERS': true,
     'PURE_CARRY_FILLERS': true,
     'RESERVER_COUNT': 3,
-    'REMOTE_MINES': true,
+    'REMOTE_MINES': 2,
```

Comment:
Could remove the level 7 definition now as it became redundant.
Unless REMOTE_MINES was interpreted as `1`, in which case the value should be 1 here.

Reply:
Nice catch, updating. I'm guessing that at controller level 4, 2 remote mines would be fine. (Maybe even earlier imo, but that's something for later) 👍